Tigran Kirakosov/flexbox_version/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Dosis:200,300,400,600&display=swap"
+      rel="stylesheet"
+    />
+    <title>Tokyo Weather Widget</title>
+  </head>
+  <body>
+    <li class="container">
+      <section class="widget">
+        <div class="widget__left-side">
+          <div class="widget__main-icon-box">
+            <svg class="widget__main-icon">
+              <use xlink:href="assets/images/sprite.svg#icon-sun" />
+            </svg>
+          </div>
+          <div class="widget__weather-desc">
+            <span>Sunny</span>
+          </div>
+          <div class="widget__weather-date">
+            <span>Tokyo - Wednesday, July 31</span>
+          </div>
+        </div>
+        <li class="widget__right-side">
+          <div class="widget__main-temperature">
+            <span>28 &deg;c</span>
+          </div>
+          <div class="widget__graph-box">
+            <picture class="widget__graph-img">

```

Comment:
afaik IE11 doesn't support `<picture>` tag https://caniuse.com/#search=picture

Reply:
Will fix that